moveit_kinematics/cached_ik_kinematics_plugin/config/kdl.yaml

```diff
@@ -0,0 +1,30 @@
+arm:
```

Comment:
This belongs in the https://github.com/ros-planning/moveit_kinematics_tests/ repo

Reply:
Again, I'm happy to do that, but note that this is not used for any unit tests. It doesn't get installed. It's only useful for development purposes.